deployments/liqo/templates/liqo-advertisement-deployment.yaml

```diff
@@ -50,9 +50,6 @@ spec:
              fieldRef:
                fieldPath: metadata.namespace
         resources:
-          limits:
-            cpu: 100m
-            memory: 50M
           requests:
             cpu: 100m
-            memory: 50M
\ No newline at end of file
+            memory: 50M
```

Comment:
Would increase the memory request, given the number of controllers it is currently running.